### PR TITLE
Harmonize vehicle rental messages

### PR DESCRIPTION
**Pull request by hbruch**
## Proposed Changes
This PR harmonizes various vehicle rental messages, e.g. by renaming bike / cycle to citybike in bikesharing context. 

Further more, it uses network configuration property `type` (which should be refactored in future to map to GBFS vehicle formFactor) to select appropriate messages.

It might make sense to add an operator name to the user facing messages as well as distinguishing between stations and free floating vehicles. This PR does not address this yet.

Note: this WIP PR has a temporary network-renaming patch in `CityBike.js` which is not intended to be merged.

## Pull Request Check List

  - A reasonable set of unit tests is included
  - Console does not show new warnings/errors
  - Changes are documented or they are self explanatory
  - This pull request does not have any merge conflicts
  - All existing tests pass in CI build
  - Code coverage does not decrease (unless measured incorrectly)

## Review

  - Read and verify the code changes
  - Test the functionality by running the UI locally with all popular browsers available in your platform
  - Check that the implementation matches the design, when such one is defined in a Jira issue
  - Merge the pull request

### CODE REVIEW

File: app/configurations/config.landstadtmobil.js [hunk 1]
```diff
@@ -160,19 +160,23 @@ export default configMerger(walttiConfig, {
                     en: 'https://www.regioradstuttgart.de/',
                 },
                 visibleInSettingsUi: false,
+                enabled: true,
+                hideCode: true,
             },
-            tier_reutlingen: {
+            'tier-reutlingen': {
```
Comment (leonardehrenfried) [inline app/configurations/config.landstadtmobil.js]: Are you sure the hypen is correct? The system id is "tier_REUTLINGEN" which is being lowercased.

**Review by leonardehrenfried — CHANGES_REQUESTED** [review]
For me the citybike icons don't show up. Do they for you?

I see the following errors in the console:
Screenshot from 2021-10-20 14-03-00

Also, there are linting errors which make the CI fail.

File: app/configurations/config.landstadtmobil.js (hunk 1 above)
Comment (leonardehrenfried) [reply app/configurations/config.landstadtmobil.js]: Screenshot from 2021-10-20 14-09-42

It's defintely tier_REUTLINGEN so you need to change this here.

Comment (hbruch) [reply app/configurations/config.landstadtmobil.js]: I tested against current prod, which was an older version. Besides this, there was an issue with upper/lower case wich is now fixed

**Pull request merged**